addons/website_event/models/event_event.py

```diff
@@ -88,6 +92,23 @@ def _default_cover_properties(self):
     start_remaining = fields.Integer(
         'Remaining before start', compute='_compute_time_data',
         help="Remaining time before event starts (minutes)")
+    # questions
+    question_ids = fields.One2many(
+        'event.question', 'event_id', 'Questions', copy=True,
+        compute='_compute_question_ids', readonly=False, store=True)
+    general_question_ids = fields.One2many(
+        'event.question', 'event_id', 'General Questions',
+        domain=[('once_per_order', '=', True)])
+    specific_question_ids = fields.One2many(
+        'event.question', 'event_id', 'Specific Questions',
+        domain=[('once_per_order', '=', False)])
+
+    @api.constrains('question_ids')
+    def _check_unique_identification_question_ids(self):
+        for event in self:
+            question_types = event.question_ids.filtered(lambda q: q.is_identification_question).mapped('question_type')
```

Comment:
Isn't it possible to have constraints based on event_id / question_type directly at sql level on question model ?